### ISSUE TITLE
"Run with IntelliJ project classpath" doesn't respect :main-opts

### ISSUE THREAD

**Issue by fjolne** [cursive-ide/cursive#2379]
"Run with Deps" respects the alias's `:main-opts` argument, allowing for simple configuration of such things as REBL, incapsulating details into the alias itself (and not in the IntelliJ run config).

"Run with IntelliJ project classpath" is often more convenient when working with a lot of aliases/forks (via "Clojure Deps" tab), but unfortunately it doesn't respect `:main-opts`.
![Screenshot 2020-05-27 at 12 34 22](https://user-images.githubusercontent.com/42907266/83003765-6fe21180-a017-11ea-8ac3-6cbcd8309079.png)

**Comment by cursive-ide** [comment]
Yes, I'm unlikely to change this since "Run with IntelliJ project classpath" by design doesn't interact with the build tool at all, and here it would have to be aware of an alias etc. Can you elaborate a bit on how it's more convenient?

**Comment by fjolne** [comment]
I often switch between published artefacts and their forks via :override-deps aliases, and then use things like "Go to definition" (which uses project classpath) to edit those forks. In case of "Run with Deps" I need to update both "Clojure Deps" tab and run config aliases every time I switch, which can easily go out of hand.

**Comment by fjolne** [comment]
OK, that's getting weird... When using "Run with IntelliJ project classpath" if I don't pass "Parameters:" then `clojure.main -r` is called. But if I pass them, then they're duplicated and `clojure.main -m fake.core -m fake.core` is called.
![Screenshot 2020-05-27 at 13 52 59](https://user-images.githubusercontent.com/42907266/83010735-652c7a00-a021-11ea-8259-877112c622e9.png)

**Comment by fjolne** [comment]
I hoped that I can pass parameters for REBL execution in run config, which would essentially solve the issue for my use cases, but it's not working.

**Comment by fjolne** [comment]
Created a separate issue for mentioned above behaviour. https://github.com/cursive-ide/cursive/issues/2380
